### PR TITLE
solution

### PR DESCRIPTION
**Pull request by uzlss**

### CODE REVIEW

File: app/players/player.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+
+class Player(ABC):
+
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:
+        pass
```
Comment (vsmutok) [inline app/players/player.py:11]: It accepts int. You must specify type which you expect from all child classes
The same in player_info

**Review by vsmutok — CHANGES_REQUESTED** [review]

File: app/players/player.py (hunk 1 above)
Comment (uzlss) [reply app/players/player.py:11]: accepts? does not it returns int so?

Comment (vsmutok) [reply app/players/player.py]: so returns 😁

File: app/players/player.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> int:
+        pass
+
+    @abstractmethod
+    def player_info(self) -> None:
```
Comment (vsmutok) [inline app/players/player.py:14]:
```suggestion
    def player_info(self) -> str:

```

**Review by vsmutok — CHANGES_REQUESTED** [review]

File: app/players/player.py (hunk 2 above)
Comment (uzlss) [reply app/players/player.py:14]: oh im retard, i thought it only printed, without return

**Review by vsmutok — APPROVED** [review]
Good 👍

**Review by Abnormaltype — APPROVED** [review]

**Pull request closed without merging**